paloalto/actions/dag_update.py

```diff
@@ -0,0 +1,52 @@
+import sys, requests, xmltodict, json, urllib3
+from string import Template
+
+from st2actions.runners.pythonrunner import Action
+
+class UpdateDAG(Action):
+       
+    def run(self, ip, firewall):
+        _result = {}
+        _uid_message = """
+        <uid-message>
+        <version>2.0</version>
+        <type>update</type>
+        <payload>
+        <register>
+        <entry ip=${ip}>
+        <tag>
+        <member>${tag}</member>
+        </tag>
+        </entry>
+        </register>
+        </payload>
+        </uid-message>
+        """
+        _tag = self.config['tag']
+        _key = self.config['api_key']
+        # print(_key, self.config['fws_ips'], _tag)                
+        xml = Template(_uid_message)
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        # for firewall in self.config['fws_ips'].split(","):
+        # print(ip, firewall)                
+        url = "https://" + firewall + "/api"
+        try:
+            response = requests.post(url + "/?type=user-id&cmd={}&key={}".
+                format(xml.substitute(ip='"{}"'.format(ip), tag=_tag), _key),verify=False, timeout=5)
```

Comment:
For better readability, i suggest formatting of the URL in separate line(s) of code above. Be consistent with string formatting, either using format or "+" but not mixing them.